types/draftjs-to-html/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for draftjs-to-html 0.8
+// Project: https://github.com/jpuri/draftjs-to-html#readme
+// Definitions by: Ivan Zverev <https://github.com/1cheese>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
+
+import { RawDraftContentState } from 'draft-js';
+
+export interface HashtagConfig {
+    trigger?: string;
+    separator?: string;
+}
+
+export default function draftToHtml(
```

Comment:
It seems like the [underlying library](https://unpkg.com/draftjs-to-html@0.8.4/lib/draftjs-to-html.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.